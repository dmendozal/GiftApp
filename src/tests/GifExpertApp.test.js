import {shallow} from "enzyme";
import React from "react";
import {GifExpertApp} from "../GifExpertApp";

describe('Prueba del componente GifExpertApp', () => {

    test('debe mostrar el componente correctamente', () => {
        const wrapper = shallow(<GifExpertApp/>);
        expect(wrapper).toMatchSnapshot();
    });

    test('debe mostrar una lista de categorias', () => {
        const categories = ['Dragon ball', 'Game of thrones']
        const wrapper = shallow(<GifExpertApp defaultCategories={categories}/>);
        expect(wrapper).toMatchSnapshot();

    })
})