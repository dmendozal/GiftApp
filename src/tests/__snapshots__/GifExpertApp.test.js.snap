// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba del componente GifExpertApp debe mostrar el componente correctamente 1`] = `
<div>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <button>
     Agregar 
  </button>
  <ol />
</div>
`;

exports[`Prueba del componente GifExpertApp debe mostrar una lista de categorias 1`] = `
<div>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <button>
     Agregar 
  </button>
  <ol>
    <GifGrid
      category="Dragon ball"
      key="Dragon ball"
    />
    <GifGrid
      category="Game of thrones"
      key="Game of thrones"
    />
  </ol>
</div>
`;
